tsconfig.json

```diff
@@ -9,6 +9,10 @@
     "experimentalDecorators": true
   },
   "exclude": [
-    "node_modules"
+    "node_modules",
+    ".vscode-test"
+  ],
+  "files": [
```

Comment:
I was referring to https://github.com/Microsoft/TypeScript/issues/3930 , seems if you set `nolib` as true you must add your own lib file as TypeScript compiler always rely on lib files to generate output. So the main purpose of `nolib` is allowing you to replace the original `lib.d.ts` to the one you want, here it's `lib.es6.d.ts`.

I'm not 100% sure whether it's correct but currently it's the only way to bypass the errors.